library/entity/entity_events/build.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
+	id("qsl.module")
+}
+
+qslModule {
+	name = "Quilt Entity Events"
+	moduleName = "entity_events"
+	id = "quilt_entity_events"
+	description = "Provides various events to do with entities"
+	version = "1.0.0"
```

Comment:
Could you merge with upstream and then fix any problems with your build so that actions pass? Notably, this line will need to be removed.